src/main/resources/index.html

```diff
@@ -419,12 +539,22 @@
             this.onPop = () => {
             };
             this.callStack = [];
-            this.firstTimestamp = -1;
+            this.firstTimestamp = 0;
             this.timestamp = -1;
             this.done = false;
             this.totalBytes = bytes.length;
             this.bytesRead = 0;
 
+            // Cache for determining if a call is a RxCall
+            this.RxCache = {};
+            // Timestamp of last log entry processed, used in Rx Ration calculation
+            this.prevRxTimestamp = -1;
+
+            // Counter of total time in Rx from start
+            this.inRxTime = 0;
```

Comment:
I like the idea of being able to aggregate metrics based on custom classification, but I don't think we should commit a one-off implementation for Rx. If we want to support this, it should be a generic implementation.

Reply:
I'd suggest leaving this out for now until we can think of how to implement generically.